databricks/koalas/tests/test_dataframe.py

```diff
@@ -1750,6 +1751,115 @@ def test_binary_operators(self):
             lambda: ks.range(10).add(ks.range(10).id),
         )
 
+    def test_binary_operator_add(self):
+        kdf = ks.DataFrame({"a": ["x"], "b": [1]})
```

Comment:
Shall we add positive tests to add string columns?

Reply:
Added a few positive tests.